Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -353,7 +353,7 @@ private func createResolvedPackages(
         )
 
         // Create target builders for each target in the package.
-        let targetBuilders = package.targets.map{ ResolvedTargetBuilder(target: $0, observabilityScope: observabilityScope) }
+        let targetBuilders = package.targets.map{ ResolvedTargetBuilder(target: $0, observabilityScope: packageObservabilityScope) }
```

Comment:
In this case there was actually an observability child scope with the metadata, it just wasn't being passed down.